ui/react-components/components/AppointmentListSidePanelSearch/AppointmentListSidePanelSearch.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import {
+  searchContainer,
+  appointmentListSidePanelSearch,
+  icon,
+  faTimesDisabled,
+  disabled,
+} from "./AppointmentListSidePanelSearch.module.scss";
+import { injectIntl } from "react-intl";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+const AppointmentListSidePanelSearch = (props) => {
+  const {
+    value,
+    onChange,
+    placeholder = "Search..",
+    style = appointmentListSidePanelSearch,
+    disabledInputSearch,
+    onClearText,
+    intl,
+  } = props;
+
+  const searchClass = classNames(style, {
+    [disabled]: disabledInputSearch,
+  });
+
+  return (
+    <div className={classNames(searchContainer)}>
+      <input
+        data-testid="appointmentSearch"
+        type="text"
+        placeholder={placeholder}
+        value={value === null ? "" : value}
+        onChange={onChange}
+        disabled={disabledInputSearch}
+        className={searchClass}
+      />
+      {(value == null || "") && (
```

Comment:
I have a doubt here about `(value == null || "")`. What exactly is intention?
Is it checking if `value is null or value is blank("")`? This statement might not do this and would end up doing something else.

Please clarify.

Reply:
I have changed it to (value == null)